internal/service/base/resource_gateway_test.go

```diff
@@ -583,7 +583,7 @@ func TestAccGateway_RADIUS(t *testing.T) {
 			resource.TestCheckNoResourceAttr(resourceFullName, "description"),
 			resource.TestCheckResourceAttr(resourceFullName, "enabled", "false"),
 			resource.TestCheckResourceAttr(resourceFullName, "type", "RADIUS"),
-			resource.TestMatchResourceAttr(resourceFullName, "radius_davinci_policy_id", verify.P1ResourceIDRegexpFullString),
```

Comment:
Is there a regex in tests for davinci ids? The convention seems to be using these regexes for ids rather than validating the specific value, so that the test doesn't rely on specific ids.